docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -1239,7 +1239,7 @@ To format a date using a string (like with the shell date command), use the "str
     {{ '%H:%M:%S' | strftime }}
 
     # Use ansible_date_time.epoch fact
-    {{ '%Y-%m-%d %H:%M:%S' | strftime(ansible_date_time.epoch) }}
+    {{ '%Y-%m-%d %H:%M:%S' | strftime(ansible_facts['date_time'['epoch']) }}
```

Comment:
Although I like `{{ '%Y-%m-%d %H:%M:%S' | strftime(ansible_facts.date_time.epoch) }}` more.
```suggestion
    {{ '%Y-%m-%d %H:%M:%S' | strftime(ansible_facts['date_time']['epoch']) }}
```

Reply:
Added the missing ]. Thanks!

For the brackets vs. dots notation I used what I saw in other places like [0]. And dot notation has some caveats, though they shouldn't be relevant to the facts generated by ansible.

[0] https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html?highlight=ansible_facts